app/controllers/projects_controller.rb

```diff
@@ -9,13 +9,12 @@ class ProjectsController < ApplicationController
   # GET /projects/new
   def new
     @project = Project.new
-    @project_image = ProjectImage.new
   end
 
   # GET /projects
   # GET /projects.json
   def index
-    @projects = Project.all
+    @projects = Project.all.select { |project| !project.attributes.hidden }
```

Comment:
Something may have been lost. Look at the latest commit made by @marcheing. The action index is a bit different.